xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 13.2.1
+JSONRPC_VERSION 13.3.1
```

Comment:
Bump should be to 13.3.0

Reply:
Yeah, I wasn't too sure although it seemed logical.  Fixed it.  Thanks.